src/main/java/com/quizapp/enterprise/config/WebSecurityAccessConfig.java

```diff
@@ -48,16 +48,21 @@ protected void configure(AuthenticationManagerBuilder auth) throws Exception {
     @Override
     protected void configure(HttpSecurity http) throws Exception {
         http.authorizeRequests()
-                .antMatchers("/quizzes").authenticated()
-                .antMatchers("/gameView/{gameId}").permitAll()
-                .anyRequest().permitAll()
+                .antMatchers("/quizzes")
+                .authenticated()
+                .antMatchers("/gameView/{gameId}")
+                .permitAll()
+                .anyRequest()
+                .permitAll()
                 .and()
                 .formLogin()
                 .usernameParameter("email")
                 .defaultSuccessUrl("/quizzes")
                 .permitAll()
                 .and()
-                .logout().logoutSuccessUrl("/").permitAll();
+                .logout()
+                .logoutSuccessUrl("/")
+                .permitAll();
```

Comment:
reformatted the streams to use indentation so the code may be cleaner and easier to read.
This recommended by the "Iterators and Streams" section of the java best practices summary referenced below.

source: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3